bitnami/jupyterhub/values.yaml

```diff
@@ -792,9 +792,9 @@ proxy:
       ##
       - ports:
           - port: {{ .Values.proxy.containerPort.http }}
-    ## @param proxy.networkPolicy.extraEgress Add extra ingress rules to the NetworkPolicy
+    ## @param proxy.networkPolicy.extraEgress Add extra egress rules to the NetworkPolicy
     ##
-    extraEgress:
+    extraEgress: ""
```

Comment:
Check `extraIngress` and `extraEgress` in this file since there are already defined values as objects (check around [here](https://github.com/bitnami/charts/blob/907669f327d7b320119ae87b1fad539d26860eca/bitnami/jupyterhub/values.yaml#L482) and [here](https://github.com/bitnami/charts/blob/907669f327d7b320119ae87b1fad539d26860eca/bitnami/jupyterhub/values.yaml#L483)).

In the same way, it is used in the same way in Redis and it's defined as an array, check https://github.com/bitnami/charts/blob/907669f327d7b320119ae87b1fad539d26860eca/bitnami/redis/values.yaml#L992

Reply:
@carrodher please check if this is what you meant: 556b793f52f6d972f33a58ce72ccc4066bc3cf7c
Changing all instances to array implies modifying some of them already set as strings (before this PR)

Reply:
Yes, I mean to be consistent between all of them, if there are 2/3 `extraEgress` or `extraIngress` parameters in the _values.yaml_ and all of them are used in the same way in the templates; all of them should have the same default value/type